src/airodump-ng/airodump-ng.c

```diff
@@ -808,6 +833,7 @@ static const char usage[] =
 	"      --ht40-               : Set channel to HT40- (802.11n)\n"
 	"      --ht40+               : Set channel to HT40+ (802.11n)\n"
 	"      --channel <channels>  : Capture on specific channels\n"
+	"      --ignore-other-chans  : Requires --channel (or -c)\n"
```

Comment:
There seems to be a misunderstanding. The first line was fine (aside from removing the comma at the end), it's just the second one that needed modification.